### PR TITLE
LOG-13588: Updates to Windows PCI pack

### PR DESCRIPTION
**Pull request by dharibabu-r7**
[LOG-13588](https://issues.corp.rapid7.com/browse/LOG-13588): As part of the JIRA, patch 1 was pushed already containing the textual updates to Windows PCI pack and added new Linux PCI pack.

This is patch 2 contains updates to the queries in Windows PCI pack.

### CODE REVIEW

**Comment by ilyabiryukov-r7** [comment]
I am not sure how to review this. :) Whoever supplied the queries should verify this.

File: windows_pci.json [hunk 1]
```diff
@@ -243,7 +243,7 @@
...
+					"statement": "where(xml.system.eventid=/4692|4693|4694|4695|4688|4696|4689|5712|4928|4929|4930|4931|4934|4935|4936|4937|4662|5169|5137|5138|5141|5136|5139|4932|4933|4668|4665|4666|4667|4818|4875|4876|4868|4869|4870|4871|4872|4873|4874|4877|4878|4879|4880|4881|4882|4883|4884|4885|4886|4887|4888|4889|4890|4891|4892|4893|4894|4895|4896|4897|4898|4899|4900|5120|5121|5122|5123|5124|5125|5126|5127|5145|5142|5144|5140|5143|5168|4664|4985/\nOR xml.system.eventid=/5051|5031|5150|5151|5154|5155|5156|5157|5158|5159|5146|5147|5152|5153|4656|4658|4690|4660|4659|4661|4663|4698|4699|5889|5890|4671|4691|4700|4701|4702|5148|5149|5888|4657|5039|4660|4659|4661|4663|5478|4960|4961|4962|4963|4965|5479|5480|5483|5484|5485|5024|5025|5030|5027|5028|5029|5032|5033|5034|5035|5037|5050|5058|5059|5071|6400|6401|6402|6403|6404|6405|6406|6407|6408|6409|6420|6421|6422|6423|6424|6416|6419|4608|4609|4616|4621|4697|4610|4611|4614|4622|4618|5038|5057|4612|4615|4816|5056|5060|5061|5062|6281|6410|6417|6418/)groupby(xml.system.eventid)"
```
Comment (dsmith-r7) [inline windows_pci.json]: is that \n meant to be there?

Comment (dharibabu-r7) [reply windows_pci.json]: Fixed it. Thanks for catching that.

**Review by dsmith-r7 — APPROVED** [review]
lgtm

**Pull request merged**